boa/src/builtins/string/mod.rs

```diff
@@ -266,6 +267,39 @@ impl String {
         }
     }
 
+    /// `String.prototype.at ( index )`
+    ///
+    /// This String object's at() method returns a String consisting of the single UTF-16 code unit located at the specified position.
+    /// Returns undefined if the given index cannot be found.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/proposal-relative-indexing-method/#sec-string-prototype-additions
```

Comment:
The correct link is:
```suggestion
    /// [spec]: https://tc39.es/proposal-relative-indexing-method/#sec-string.prototype.at
```